ConscryptMode.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Conscrypt Mode
+group: User Guide
+order: 2
+redirect_from: "/activity-lifecycle/"
+---
+
+# ConscryptMode 
+
+Robolectric can currently either use Conscrypt and BouncyCastle or just BouncyCastle as the security provider. 
+In order to migrate tests over time, there is a `ConscryptMode` annotation that controls whether Conscrypt is loaded as the default security provider with BouncyCastle as backup. 
+
+### Implementation
```

Comment:
It's better we just link them to it since this code could change and then we'd have a difference. Above we say `ConscryptMode`. We could just do [ConscryptMode](LinkToThatCode) instead.